samples/features/azure-arc/deployment/kubeadm/ubuntu-single-node-vm/setup-controller.sh

```diff
@@ -335,7 +335,7 @@ echo "Starting to deploy azdata cluster..."
 #
 azdata arc dc config init -s azure-arc-kubeadm-private-preview -t azure-arc-custom --force
 
-azdata arc dc create -n $ARC_DC_NAME -c azure-arc-custom --namespace $CLUSTER_NAME --location $ARC_DC_REGION --resource-group $ARC_DC_RG --subscription $ARC_DC_SUBSCRIPTION --connectivity-mode indirect
+azdata arc dc create -n $ARC_DC_NAME -p azure-arc-kubeadm-private-preview --namespace $CLUSTER_NAME --location $ARC_DC_REGION --resource-group $ARC_DC_RG --subscription $ARC_DC_SUBSCRIPTION --connectivity-mode indirect
```

Comment:
Changing from -c to -p should be enough. no need to change the profile name from azure-arc-custom to azure-arc-kubeadm-private-preview.

Reply:
"azure-arc-custom"  is not supported.
    --profile-name -p              : The name of an existing configuration profile. One of the
                                     following: ['azure-arc-eks-private-preview', 'azure-arc-
                                     openshift-private-preview', 'azure-arc-kubeadm-private-
                                     preview', 'azure-arc-aks-private-preview'].